services/integration/src/controllers/constitution.controllers/constitution.js

```diff
@@ -1,18 +1,22 @@
-const Constitution = require('../models/constitution.model');
-const { asyncHandler } = require('../utils');
+const Constitution = require('../../models/constitution/constitution.models');
```

Comment:
why you are naming this file (`constituion.models`) like that? Are you defining more than one model inside that file?